cpp/src/column/column.cu

```diff
@@ -270,7 +267,8 @@ struct create_column_from_view_vector {
 // Copy from a view
 column::column(column_view view, cudaStream_t stream,
                rmm::mr::device_memory_resource *mr) :
-column( *cudf::experimental::type_dispatcher(view.type(), create_column_from_view{view, stream, mr})) {}
+  column{std::move(*experimental::type_dispatcher(view.type(),
```

Comment:
Please add a comment here stating that std::move is indeed required, so nobody reverts the change later on.

Reply:
Done